### PR TITLE
Migrate parquet account txn processor to sdk

### PR DESCRIPTION
**Pull request by yuunlimm**
### Description
migrating parquet account txn processor to sdk

### Test Plan
mroe rows in parquet is expected
Screenshot 2024-12-11 at 4 19 37 PM
Screenshot 2024-12-11 at 4 14 03 PM

### CODE REVIEW

**Comment by yuunlimm** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/626?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#626** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/626?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/626?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#625** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/625?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#619** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/619?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#616** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/616?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#615** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/615?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: rust/processor/src/db/common/models/account_transaction_models/raw_account_transactions.rs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+impl RawAccountTransaction {
+    /// This table will record every transaction that touch an account which could be
+    /// a user account, an object, or a resource account.
+    /// We will consider all transactions that modify a resource or event associated with a particular account.
+    /// We will do 1 level of redirection for now (e.g. if it's an object, we will record the owner as account address).
+    /// We will also consider transactions that the account signed or is part of a multi sig / multi agent.
+    /// TODO: recursively find the parent account of an object
+    /// TODO: include table items in the detection path
```
Comment (dermanyang) [inline rust/processor/src/db/common/models/account_transaction_models/raw_account_transactions.rs:35]: Are these TODOs still relevant?

**Comment by dermanyang** [comment]
did you do a sanity check on number of rows in the equivalent version range on the respective legacy processor?

**Review by dermanyang — APPROVED** [review]

File: rust/processor/src/db/common/models/account_transaction_models/raw_account_transactions.rs (hunk 1 above)
Comment (dermanyang) [reply rust/processor/src/db/common/models/account_transaction_models/raw_account_transactions.rs:35]: Just for context, they're from this original PR: https://github.com/aptos-labs/aptos-core/pull/9076

**Comment by yuunlimm** [comment]
### Merge activity

* **Dec 11, 10:27 PM EST**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/626).

**Pull request merged**